Apply needed fixes and improvements (#90)
Hydra-Chain/hydragon-core-contracts: docs/HydraChain/HydraChain.md

function initiateBanThreshold() external view returns (uint256)
```

> Validator inactiveness (in blocks) threshold that needs to be reached or passed to initiate ban for a validator

Reviewer: ```suggestion
Threshold for validator inactiveness (in blocks). A ban can be initiated for a validator if this threshold is reached or exceeded.
```
